api/docker-compose.yaml

```diff
@@ -28,6 +28,10 @@ x-api-defaults: &api-defaults
 services:
   scripts:
     <<: *api-defaults
+  ports-scripts:
+    <<: *api-defaults
+    ports:
+      - 127.0.0.1:3307:3307
```

Comment:
Oops sorry - saw this and then got meeting'd.

This is fine for my setup. But I'm curious why 3307 here rather than the existing container on 3306?